e2e/tests/attendance-tests.spec.ts

```diff
@@ -1,360 +1,278 @@
 import { expect, test } from "#e2e/fixtures.ts";
 
-test.describe("Attendance Module", () => {
-  const FIXED_DATE = "2025-01-23";
+test.only("Record attendance for one activity", async ({ page }) => {
+  await page.getByRole("navigation").getByText("Attendance").click();
+  await page.getByRole("button", { name: "Record" }).click();
 
-  test.beforeEach(async ({ page }) => {
-    await page.clock.install({ time: FIXED_DATE });
-  });
+  const dateField = page.getByLabel("Date");
+  await expect(dateField).toHaveValue("1/23/2025");
 
-  test("Record attendance for one activity", async ({ page }) => {
-    await page.getByRole("navigation").getByText("Attendance").click();
-    await page.getByRole("button", { name: "Record" }).click();
+  await dateField.fill("12/15/2024");
 
-    /*
-      Verify the date field displays the current date
-    */
-    const fixedDate = new Date(FIXED_DATE);
-    const formattedDateRegex = new RegExp(
-      `${fixedDate.getMonth() + 1}/${fixedDate.getDate()}/${fixedDate.getFullYear()}|` +
-        `${(fixedDate.getMonth() + 1).toString().padStart(2, "0")}/${fixedDate.getDate().toString().padStart(2, "0")}/${fixedDate.getFullYear()}`,
-    );
-    const dateField = page.getByLabel("Date");
-    await expect(dateField).toHaveValue(formattedDateRegex);
+  // FIXME: A simple .click() does not trigger the action and we don’t know why.
+  await page.getByText("School Class 2B").dispatchEvent("click");
 
-    /*
-      Allow backdated editing for the date field
-    */
-    const backdatedDate = "12/15/2024";
-    await dateField.fill(backdatedDate);
-    await expect(dateField).toHaveValue(backdatedDate);
+  await expect(
+    page.getByRole("heading", { name: "School Class 2B" }),
+  ).toBeVisible();
 
-    /*
-      Verify list of classes is displayed
-    */
-    await page.waitForSelector("mat-card-header");
-    const classList = page.locator("mat-card-header");
-    const count = await classList.count();
-    expect(count).toBeGreaterThan(0);
```

Comment:
We’re testing an artifact of the implementation (the class list) here. I removed this because we should focus on things that are directly visible to the user and I don’t know what this is in this case.